siliconcompiler/core.py

```diff
@@ -2632,7 +2614,7 @@ def summary(self, steplist=None, show_all_indices=False):
         if self.get('mode') == 'asic':
             info_list.extend(["foundry : " + self.get('pdk', 'foundry'),
                               "process : " + self.get('pdk', 'process'),
-                              "targetlibs : "+" ".join(self.get('asic', 'targetlib'))])
+                              "targetlibs : "+" ".join(self.get('asic', 'logiclib'))])
```

Comment:
Nit: make it clear that the `+` character is an operator, not a string?

```suggestion
                              "targetlibs : " + (" ".join(self.get('asic', 'logiclib')))])
```